_inc/client/at-a-glance/index.jsx

```diff
@@ -77,29 +78,33 @@ class AtAGlance extends Component {
 					}
 					<div className="jp-at-a-glance__item-grid">
 						<div className="jp-at-a-glance__left">
-							<DashProtect { ...settingsProps } />
+							<DashActivity { ...settingsProps } />
 						</div>
 						<div className="jp-at-a-glance__right">
-							<DashScan { ...settingsProps } siteRawUrl={ this.props.siteRawUrl } />
+							<DashProtect { ...settingsProps } />
```

Comment:
Looks like `this.props.siteRawUrl` is empty (`""` ) at this point. Probably it wasn't being used and/or passed down from its parent.

Reply:
That's odd @oskosk I'm seeing this defined at all times. I added a console.log at the beginning of render() and it shows the function perfectly.

Reply:
weird...